user/build-environment-updates/2016-12-01.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Build Environment Update History
+layout: en
+permalink: /user/build-environment-updates/2016-12-01/
+category: build_env_updates
+date: 2016-12-01
+---
+
+## 2016-12-01
+
+### Schedule
+
+[2016-12-01 15:00 UTC](http://everytimezone.com/#2016-12-1,180,cn3)
+
+### Platforms
+
+This update applies to fully-virtualized Ubuntu Trusty and Precise.
```

Comment:
The term we are using in https://docs.travis-ci.com/user/ci-environment/#Virtualization-environments is "sudo-enabled".